service/src/main/java/org/cbioportal/genome_nexus/service/remote/RevueDataFetcher.java

```diff
@@ -0,0 +1,30 @@
+package org.cbioportal.genome_nexus.service.remote;
+import org.cbioportal.genome_nexus.model.VuesJsonRecord;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+
+@Component
+public class RevueDataFetcher {
+
+    private static final String revueUrl = "https://raw.githubusercontent.com/knowledgesystems/reVUE-data/main/VUEs.json";
```

Comment:
@leexgh should this be some configurable property? E.g. if we want revue-beta.genomenexus.org to point to a dev version of the reVUE JSON? And maybe we could have some property whether or not to include validated VUEs only or all VUEs?

Reply:
Yes, I added two properties: `revue.url` and `overwrite_by_confirmed_revue_only`